documentation/Docker-metrics.md

```diff
@@ -1,6 +1,6 @@
 # Metrics
 
-The Docker deployment includes a metrics collection system which is available on production as well as development/test environments. It consists of a Prometheus metrics server which scrapes metrics from various exporters. Grafana is provided as frontend to visualise metrics and create graphs/dashboards.
+The Docker deployment includes a metrics collection system which is available on production as well as development/test environments. It consists of a Prometheus metrics server which scrapes metrics from various exporters. Grafana is provided as frontend to visualise metrics and create graphs/dashboards. The various exporters, Prometheus and Grafana containers can be disabled by not loading the `monitoring` profile, by overriding the default profiles with e.g. `COMPOSE_PROFILES=cron` in `local.env`.
```

Comment:
I think we should refrain from giving these kinds of instructions for COMPOSE_PROFILES completely (for which I'm guilty of as well). And go with @mxsasha idea https://github.com/internetstandards/Internet.nl/pull/1589#issuecomment-2529669134 and have 1 place in the documentation to refer to with a table with all possible permutations of COMPOSE_PROFILES (that are sane at least) and have them copy pasteable so there is no ambiguity.